wandering-warriors/main.py

```diff
@@ -1,30 +1,54 @@
 from kivy.app import App
-from kivy.uix.widget import Widget
 from kivy.lang import Builder
+from kivy.uix.screenmanager import ScreenManager, Screen
+from kivy.uix.widget import Widget
 
 
-class ToolManager(Widget):
+class Calculator(Screen):
     pass
 
 
-class Abacus(Widget):
+class Settings(Screen):
     pass
 
 
-class CuneiformDrawingInput(Widget):
+class TopMenu(Widget):
+    pass
+
+
+class Abacus(Widget):
     pass
 
 
 class Ledger(Widget):
     pass
 
 
-class AbacusApp(App):
+class OperationsBar(Widget):
```

Comment:
General comment for the classes: could we have docstrings? That seems to be something the judges look for (at least in the last code jam I remember them talking about how docstrings are very nice to have).

Reply:
I don't think they're necessary, especially not for widget classes